backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -34,12 +34,44 @@
 from tracim_backend.models.data import ContentRevisionRO
 from tracim_backend.models.data import UserRoleInWorkspace
 from tracim_backend.models.data import Workspace
+from tracim_backend.views.search_api.schemas import SearchFilterQuerySchema
 
 FILE_PIPELINE_ID = "attachment"
 FILE_PIPELINE_SOURCE_FIELD = "b64_file"
 FILE_PIPELINE_DESTINATION_FIELD = "file_data"
 
 
+class AdvancedSearchParameters:
```

Comment:
It's related to content Search, we should probably adapt the name according to that.

Reply:
Indeed, thanks